# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/webcodecs/video-encoder-config.https.any.js
# Commit: 45b561797261
# Full Hash: 45b561797261b18d83e66b09c3bf9be9820e4d09
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-03-11 21:13:39
# Description:
#   Bug 1878300 - Only accept vp8 for vp8 codec string. r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D200678
# ==============================================================================

diff -r 6901b5e67c87 -r 45b561797261 testing/web-platform/tests/webcodecs/video-encoder-config.https.any.js
--- a/testing/web-platform/tests/webcodecs/video-encoder-config.https.any.js	Mon Mar 11 14:01:17 2024 +0000
+++ b/testing/web-platform/tests/webcodecs/video-encoder-config.https.any.js	Mon Mar 11 14:16:41 2024 +0000
@@ -90,6 +90,14 @@
     },
   },
   {
+  comment: 'VP8 codec string not accepted by Web Codecs',
+    config: {
+      codec: 'vp08.00.10.08',
+      width: 640,
+      height: 480,
+    },
+  },
+  {
     comment: 'Codec with bad casing',
     config: {
       codec: 'vP8',
